test/test.js

```diff
@@ -239,98 +239,19 @@ test('normal, error-less state activation flow for two states', function(t) {
 			t.equal(context.parameters.wat, 'some value', 'got the the parent\'s parameter value in the child2\'s activate function')
 
 			t.end()
-		}
+		},
 	})
 
 	stateRouter.go('parent.child1', { wat: 'some value' })
 })
 
-test('stateIsActive', function(t) {
-	var stateRouter = getTestState(t).stateRouter
-
-	t.plan(6)
-
-	stateRouter.addState({
-		name: 'parent',
-		template: '',
-		route: '/parent',
-	})
-
-	stateRouter.addState({
-		name: 'parent.child1',
-		template: '',
-		route: '/child1'
-	})
-
-	stateRouter.addState({
-		name: 'parent.child2',
-		template: '',
-		route: '/child2',
-	})
-
-	stateRouter.on('stateChangeEnd', function() {
-		t.ok(stateRouter.stateIsActive('parent'), 'parent is active')
-		t.ok(stateRouter.stateIsActive('parent.child1'), 'parent.child1 is active')
-		t.notOk(stateRouter.stateIsActive('parent.child2'), 'parent.child2 is not active')
-		t.notOk(stateRouter.stateIsActive('not a real state'), 'non-existant state is not active')
-
-		t.notOk(stateRouter.stateIsActive('parent.child1', { butts: 'no' }), 'shouldn\'t match wuth butts=no')
-		t.ok(stateRouter.stateIsActive('parent.child1', { butts: 'yes' }), 'should match with butts=yes')
-
-		t.end()
-	})
-
-	stateRouter.go('parent.child1', { butts: 'yes' })
-})
-
-test('stateIsActive but states with that substring are not', function(t) {
-	var stateRouter = getTestState(t).stateRouter
 
-	t.plan(4)
+test('evaluateCurrentRoute with url set', t => {
+	const testState = getTestState(t)
+	const stateRouter = testState.stateRouter
+	const hashRouter = testState.hashRouter
 
-	stateRouter.addState({
-		name: 'parent',
-		template: '',
-		route: '/parent',
-	})
-
-	stateRouter.addState({
-		name: 'parent-thing',
-		template: '',
-		route: '/parent-thing',
-	})
-
-	stateRouter.addState({
-		name: 'parent.child',
-		template: '',
-		route: '/child'
-	})
-
-	stateRouter.addState({
-		name: 'parent.child-thing',
-		template: '',
-		route: '/child-thing',
-	})
-
-	stateRouter.on('stateChangeEnd', function() {
-		t.ok(stateRouter.stateIsActive('parent'), 'parent is active')
-		t.notOk(stateRouter.stateIsActive('parent-thing'), 'parent-thing is not active')
-
-		t.notOk(stateRouter.stateIsActive('parent.child'), 'parent.child is active')
-		t.ok(stateRouter.stateIsActive('parent.child-thing'), 'parent.child-thing is not active')
-
-		t.end()
-	})
-
-	stateRouter.go('parent.child-thing', { butts: 'yes' })
-})
```

Comment:
These two tests were moved to a different/their own test file?

Reply:
Yeah, related to #112, I wanted to add a new test for `stateIsActive`, and it made sense to move these tests to their own file at that point.